### DISCUSSION TITLE
Is it possible to stop the hotkey (used to invoke a script) to go further in the program stack?

### DISCUSSION THREAD

**Discussion by mikkovedru** [autokey/autokey#627] [Q&A]
There is a hotkey I want to use. Unfortunately, it is also used in OS, in other programs (like a browser) and sites (like youtube and quora). Often, it's impossible to turn those stolen default hotkeys off or to remap them to something else.

The hotkey and the script work fine and do what I want. Unfortunately after this, the key combination goes forward and invokes additional unwelcomed action by website/program/OS.

I couldn't find a way to do it in Autokey.

Am I not seeing it or does this feature not exist? Is it realistic to long for such a feature? Should I create a separate suggestion issue?

**Comment by josephj11** [comment]
Haven't see this behavior before. Usually, AutoKey consumes the keypresses or something else does and AutoKey never sees them. We would need [full details](https://github.com/autokey/autokey-python2/wiki/Problem-Reporting-Guide) to pursue this.

You could file an issue here, but it might be better to hop onto our advanced support over on [Gitter](https://gitter.im/autokey/autokey) first to see if we can identify the issue more precisely.
